UPSTREAM_UPGRADE.md

```diff
@@ -39,3 +39,54 @@ $ grep -A 1 storageClass values.schema.json
                         "storageClass": {
                             "type": ["string", "null"]
 ```
+
+## Checklist of GS custom changes
+
+Here is a list of Giantswarm changes on these charts.
+Please note that this list is not exhaustive. It's rather to be seen as the set of minimal updates to do.
+
+- kube-linter setup (`.kube-linter.yaml` file)
+- Chart
+  - home: https://github.com/giantswarm/xxx-app
+  - icon: https://s.giantswarm.io/app-icons/xxx
+  - maintainers: add giantswarm
+  - url: giantswarm
+  - sources: giantswarm
+  - version: `[[ .Version ]]`
```

Comment:
I'd add the Chart annotations (especially the team one)

Reply:
Done.